build/README.md

```diff
@@ -29,10 +29,40 @@ Build with different supported versions of Spark, by default using Spark 2.2.1 t
 ```
 mvn -DskipTests -Pspark-2.1 -Dspark.version=2.1.0 clean package
 mvn -DskipTests -Pspark-2.2 -Dspark.version=2.2.1 clean package
+mvn -DskipTests -Pspark-2.3 -Dspark.version=2.3.2 clean package
 ```
 
 Note: If you are working in Windows environment, remember to add `-Pwindows` while building the project.
 
+## MV Feature Build
+Add mv module and sourceDirectory to the spark profile corresponding to the parent pom.xml file and recompile.
+The compile command is the same as the command in the previous section
+```
+    <profile>
```

Comment:
Do we really need this?
Currently we can use `-Pmv` to include the MV feature while compiling.